semcore/utils/src/findComponent.ts

```diff
@@ -6,27 +6,34 @@ import getOriginChildren from './getOriginChildren';
 function findComponent(
   Children: any,
   names: string[],
+  recursively?: boolean,
```

Comment:
Could you add some basic recursion depth protection?

Reply:
added